packages/stencil/components/button/src/components/osds-button/osds-button.scss

```diff
@@ -45,6 +44,27 @@
   }
 }
 
+
+:host(:not([inline])) {
```

Comment:
why not use directly `:host` and remove unnecessary selector ?

Reply:
(same question for all of course)

Reply:
Done